cmd/jsonnet.cpp

```diff
@@ -673,38 +685,76 @@ int main(int argc, const char **argv)
 
             case FMT: {
                 std::string output_file = config.outputFile;
-                if (config.fmtInPlace) {
-                    if (config.inputFile == "-") {
-                        std::cerr << "ERROR: Cannot use --in-place with stdin" << std::endl;
-                        jsonnet_destroy(vm);
-                        return EXIT_FAILURE;
+
+                if (config.fmtInPlace || config.fmtTest) {
+                    assert(config.inputFiles.size() >= 1);
+                    for (std::string inputFile: config.inputFiles) {
+                        if (config.fmtInPlace) {
+                            output_file = inputFile;
+
+                            if (inputFile == "-") {
```

Comment:
I realize you copied this restriction from the existing code, but why does it exist @sparkprime? Seems useful to pipe files through the formatter rather than modify them in place.